api/cluster.go

```diff
@@ -66,9 +66,10 @@ type ClusterInfo struct {
 // PeerInfo shows information about all the peers in the cluster that
 // are supporting the stream or consumer.
 type PeerInfo struct {
-	Name    string        `json:"name" yaml:"name"`
-	Current bool          `json:"current" yaml:"current"`
-	Offline bool          `json:"offline,omitempty" yaml:"offline"`
-	Active  time.Duration `json:"active" yaml:"active"`
-	Lag     uint64        `json:"lag,omitempty" yaml:"lag"`
+	Name     string        `json:"name" yaml:"name"`
+	Current  bool          `json:"current" yaml:"current"`
+	Observer bool          `json:"observer" yaml:"observer"`
```

Comment:
I noticed you did omitempty in json on the nats.go so we should probably do the same here? Assuming thats whats in the server.  But not on yaml.

Reply:
Yes good catch, fixed.